src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -622,6 +622,15 @@ public String getAuthorTitleYear(int maxCharacters) {
     }
 
     /**
+     * Returns the title of the given BibTeX entry as an optional.
+     *
+     * @return the title of a BibTeX entry or null if no title was found.
+     */
+    public Optional<String> getTitle() {
+        return getField(FieldName.TITLE);
```

Comment:
Yes, that was careless of me 😄.
I have updated the java doc for `getTitle` to reflect what it actually does.

I was aware that `bes` contained all the selected `BibEntry` objects, but left the naming like that to be consistent with the other naming. 
I have changed the variable names to be more descriptive.